### DISCUSSION TITLE
Tailwind sibling with other pseudo chain dont work

### DISCUSSION THREAD

**Discussion by devopg** [tailwindlabs/tailwindcss#7301] [Help]
hi i want apply two pseudo selectors by tailwind

            <div>
                <input class="peer" type="checkbox" checked="checked" />
                <label class="peer-checked:before:bg-green-500"></label>
            </div>
but its dont work.

How can i apply sibling & before both?

ps. if i test apply only peer or before separately all work fine!

**Comment by adamwathan** [comment]
In the code you've shared the reason you don't see anything is because the pseudo element has no height or width so it's effectively invisible. The styles are being applied though.

If I make the pseudo-element a block element and give it a size you can see it's there:

https://play.tailwindcss.com/K8Wj3BHFFH
